src/Cake.Prca.PullRequests.Tfs/TfsPullRequestSystem.cs

```diff
@@ -98,6 +106,34 @@ public TfsPullRequestSystem(ICakeLog log, TfsPullRequestSettings settings)
                 this.pullRequest.SourceRefName);
         }
 
+        /// <summary>
+        /// Votes for the pullrequest.
+        /// </summary>
+        /// <param name="vote">The vote for the pull request.</param>
+        public void Vote(TfsPullRequestVote vote)
+        {
+            if (!this.ValidatePullRequest())
+            {
+                return;
+            }
+
+            Identity authorizedIdenity;
+            using (var gitClient = this.CreateGitClient(out authorizedIdenity))
+            {
+                var request =
+                    gitClient.CreatePullRequestReviewerAsync(
+                        new IdentityRefWithVote() { Vote = (short)vote },
+                        this.pullRequest.Repository.Id,
+                        this.pullRequest.PullRequestId,
+                        authorizedIdenity.Id.ToString(),
+                        CancellationToken.None);
+
+                var createdReviewer = request.Result;
```

Comment:
I think for the time being it's okey to throw errors. The caller could handle them, furthermore it's consistent with the current overall implementation.